CONTRIBUTING.md

```diff
@@ -53,12 +53,12 @@ MAVEN_OPTS='--add-opens java.base/java.lang=ALL-UNNAMED --add-opens java.base/ja
 ### Running the Yarn frontend build
 
 > [!TIP]
-> If you already have Node.js installed, you do not need to change your path. Start using `yarn` by enabling [Corepack](https://yarnpkg.com/corepack) with `corepack enable`, if it isn't already; this will add the `yarn` binary to your PATH.
+> If you already have Node.js 20.x LTS installed, you do not need to change your path. Start using Yarn by enabling [Corepack](https://yarnpkg.com/corepack) with `corepack enable`, if it isn't already; this will add the `yarn` binary to your path.
```

Comment:
Clarifying that we only support Node.js 20.x LTS.

I changed the first occurence of "`yarn`" to "Yarn" because this is a reference to the Yarn project, not the `yarn` binary. Even though we later use "`yarn` binary" in the same sentence, this is not inconsistent because the first usage refers to the project and the second usage refers to the binary.

Elsewhere in the sentence we replace "PATH" with "path" for consistency with the beginning of the sentence and the rest of the document. While both "`PATH`" (as a particular variable name) or "path" (referring to the general concept of a search path) would be correct here, this document generally uses the latter, so be consistent with usage elsewhere in this sentence and in this document.

Reply:
> Clarifying that we only support Node.js 20.x LTS.

I am not sure if that statement is true. If that is what we want we should then change the `package.json` to add 

```
"engines": {
  "node": ">=20.0.0
},
```

Reply:
with this change do we need this sentence? (as nothing will automatically update this.

Reply:
> with this change do we need this sentence?

I suppose not, since Yarn [apparently enforces the version specified by `engines`](https://chrisshepherd.me/posts/enforcing-nodejs-npm-yarn-versions)—unlike NPM. I haven't tested this, though. I don't feel strongly about discussing the issue further, so I have simply removed the change in commit c6a69c8908. With the change now removed, there ought not to be a reason to discuss the issue further, either.